signals_test.go

```diff
@@ -20,9 +20,7 @@ import (
 	"time"
 )
 
-var (
-	SLEEPIT, _ = filepath.Abs("./bin/sleepit")
-)
+var SLEEPIT, _ = filepath.Abs("./bin/sleepit")
```

Comment:
While you are simply applying a gofumpt on this.

I'm surprised to see:
- the error is ignored
- this variable name is in uppercase
- the variable is apparently used only in one test.

I would expect this variable to be used only where needed and to check the possible error